### PR TITLE
Initializing repo

### PR DESCRIPTION
**Pull request by alexanderConstantinescu**
This commit initializes the repository 

It adds:
- OWNERS file
- Makefile
- Dockerfile
- .gitignore
- Base file for binary

/assign @abhat @danwinship @tssurya @squeed

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8 AS builder
```
Comment (danielmellado) [inline Dockerfile:1]: I'd also add a rhel7 builder, which might be needed later on

i.e.

`FROM registry.ci.openshift.org/ocp/builder:rhel-7-golang-1.15-openshift-4.7 AS rhel7`

Comment (alexanderConstantinescu) [reply Dockerfile:1]: I don't think it will. This component will only be deployed on OCP >= 4.8 which only runs rhel8.

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - abhat
+  - danwinship
+  - dcbw
+  - knobunc
+  - trozet
+  - rcarrillocruz
+  - squeed
+
+# Bugzilla info
+component: Networking
```
Comment (danwinship) [inline OWNERS:24]: I think you need to specify subcomponent as well. Even though there isn't a correct answer. Pick either openshift-sdn or ovn-kubernetes

Comment (alexanderConstantinescu) [reply OWNERS:24]: Done

**Comment by alexanderConstantinescu** [comment]
@openshift/openshift-team-sdn PTAL

**Comment by tssurya** [comment]
/lgtm

Would be nice to have a brief README asap.

**Pull request merged**